Dockerfile.kernel

```diff
@@ -1,7 +1,7 @@
 # Copyright (c) Jupyter Development Team.
 # Distributed under the terms of the Modified BSD License.
 
-FROM jupyter/pyspark-notebook:0017b56d93c9
+FROM jupyter/all-spark-notebook:0017b56d93c9
 
 RUN pip install jupyter_kernel_gateway==0.5.0
 # install ipywidgets 5.x
```

Comment:
For first one, it's something like:

`RUN /opt/conda/envs/python2/bin/pip install jupyter_declarativewidgets` 

Check the path (should be in the docker-stacks README) and probably add a version too.